agent/Makefile

```
@@ -1,7 +1,15 @@
-TEMPDIR ?= .tmp
+# Go関連のターゲット
+.PHONY: test test-coverage clean
 
-$(TEMPDIR):
-	mkdir -p $(TEMPDIR)
+# テストの実行
+test:
+	go test ./... -v
 
-build: $(TEMPDIR)
-	cd cmd/runner && go build -o ../../$(TEMPDIR)/build/agent
+# テストカバレッジの計測と結果の出力
+test-coverage:
+	go test ./... -coverprofile=coverage.out
+	go tool cover -html=coverage.out -o coverage.html
```

from test_quality_reviewer
テストカバレッジの計測は実装されていますが、最小カバレッジ率の閾値が設定されていません。また、カバレッジレポートの保存先ディレクトリの指定も推奨します。

```suggestion
test-coverage:
	go test ./... -coverprofile=coverage.out -covermode=atomic -coverpkg=./...
	go tool cover -html=coverage.out -o reports/coverage.html
	@if [ "$$(go tool cover -func=coverage.out | grep total | awk '{print $$3}' | sed 's/%//')" -lt 80 ]; then \
		echo "Test coverage is below 80%" && exit 1; \
	fi
```